A3A/addons/core/Templates/AddonVics.hpp

```diff
@@ -11,6 +11,116 @@ class AddonVics
+        files[] = { {"Civ", "vanilla_Civ.sqf"} };
+        displayName = "Vanilla vehicle pack";
+        description = "Vanilla vehicle pack";
+        loadedMessage = "Vanilla loaded, civilian vehicle pool expanded";
+    };
+
+    class Apex
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"A3_Soft_F_Exp_Offroad_02"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "apex_Civ.sqf"} };
+        displayName = "Apex vehicle pack";
+        description = "Apex vehicle pack(civillian vehicles (don't forget to enable dlc above!))";
+        loadedMessage = "Apex loaded, civilian vehicle pool expanded";
+    };
+
+    class Lawsofwar
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"A3_Soft_F_Orange_Van_02"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "lawsofwar_Civ.sqf"} };
+        displayName = "Laws of war vehicle pack";
+        description = "Laws of war vehicle pack(if you want only civilian vehicles from dlc(don't forget to enable dlc above!))";
+        loadedMessage = "Lawsofwar loaded, civilian vehicle pool expanded";
+    };
+
+    class Contact
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"A3_Soft_F_Enoch_Offroad_01"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "contact_Civ.sqf"} };
+        displayName = "Contact vehicle pack";
+        description = "Contact vehicle pack(if you want only civilian vehicles from dlc(don't forget to enable dlc above!))";
+        loadedMessage = "Contact loaded, civilian vehicle pool expanded";
+    };
+
+    class Kart
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"A3_Soft_F_Kart_Kart_01"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "kart_Civ.sqf"} };
+        displayName = "Kart(kart)";
+        description = "Kart";
+        loadedMessage = "Kart";
+    };
+
+    class WesternSahara
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"Vehicles_F_lxWS_Offroad_01"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "westernsahara_Civ.sqf"} };
+        displayName = "Western Sahara vehicle pack";
+        description = "Western Sahara vehicle pack(if you want only civilian vehicles from cdlc(don't forget to enable cdlc above!))";
+        loadedMessage = "Western Sahara loaded, civilian vehicle pool expanded";
+    };
+
+    class ReactionForces
+    {
+        path = QPATHTOFOLDER(Templates\AddonVics);
+        requiredAddons[] = {"RF_Vehicles_Pickup_01"};
+        //format {side, file relative to path}
+        files[] = { {"Civ", "reactionforces_Civ.sqf"} };
+        displayName = "Reaction Forces vehicle pack";
+        description = "Reaction Forces vehicle pack(if you want only civilian vehicles from cdlc(don't forget to enable cdlc above!))";
```

Comment:
This will probably look awful to read.
```suggestion
        description = "Reaction Forces Vehicle Pack (Civ Vehicles)";
```

Reply:
It looks fine in game, but I can delete "if you want only civilian vehicles from cdlc" part, and change "vehicle pack" to "civilian vehicle pack"

Reply:
that would work, just make sure to capitalise it